src/query/storage/m3/consolidators/multi_fetch_tags_result.go

```diff
@@ -35,10 +35,11 @@ type multiSearchResult struct {
 	meta      block.ResultMetadata
 	err       xerrors.MultiError
 	seenIters []client.TaggedIDsIterator // track known iterators to avoid leaking
+	// FIXME: take a series iter version instead.
```

Comment:
Do we need to address this before this PR is merged?

Reply:
Ah wait this was inaccurate, since this stuff works on tags only, it's immune to mismatch ID issues